server/app/controllers/admin/AdminProgramController.java

```diff
@@ -65,6 +69,11 @@ public AdminProgramController(
   @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
   public Result index(Request request) {
     Optional<CiviFormProfile> profileMaybe = profileUtils.currentUserProfile(request);
+    // TODO(#1238): Remove the old view once the new rendering
+    // should be default.
+    if (request.queryString().containsKey("v2")) {
```

Comment:
Note to reviewer: The rationale here is that we can get this in front of a program admin to test live and iterate on the design. If there's a feature miss or issue, they needn't be blocked while we iterate.